src/web/screens/manageParticipants.tsx

```diff
@@ -0,0 +1,41 @@
+import { Suspense } from 'react';
+import { Await, defer, useLoaderData } from 'react-router-dom';
+
+import { ParticipantRequestDTO } from '../../api/participantsRouter';
+import { Loading } from '../components/Core/Loading';
+import { ParticipantRequestsTable } from '../components/ParticipantRequests/ParticipantRequestsTable';
+import { GetParticipantsAwaitingApproval } from '../services/participant';
+import { PortalRoute } from './routeUtils';
+
+function ManageParticipants() {
+  const { participantsAwaitingApproval } = useLoaderData() as {
+    participantsAwaitingApproval: ParticipantRequestDTO[];
+  };
+
+  return (
+    <div>
+      <h1>Manage Participants</h1>
+      <p>View and manage UID2 Portal participant requests and information.</p>
+      <h2>Participant Requests</h2>
+      <Suspense fallback={<Loading />}>
+        <Await resolve={participantsAwaitingApproval}>
+          {(participantRequests: ParticipantRequestDTO[]) => (
+            <ParticipantRequestsTable participantRequests={participantRequests} />
+          )}
+        </Await>
+      </Suspense>
+    </div>
+  );
+}
+
+export const ManageParticipantsRoute: PortalRoute = {
+  path: '/dashboard/manageParticipants',
+  description: 'Manage Participants',
+  element: <ManageParticipants />,
+  loader: async () => {
+    const participantsAwaitingApproval = await GetParticipantsAwaitingApproval();
```

Comment:
could you remove the await from here, otherwise this will wait for the request to resolve and then render the page

Reply:
here is the documentation: https://reactrouter.com/en/main/route/loader

Reply:
done! so I guess it only makes sense to await subsequent calls (rather than the first). e.g. sharingPermissions has:
```
  loader: async () => {
    const participants = GetAllAvailableParticipants();
    const participantTypes = await GetAllParticipantTypes();
    return defer({ participants, participantTypes });
  },
```
so the page loads after participants comes back, then we async load participantTypes.
Is that right?